### DISCUSSION TITLE
recovered: kv store with key <nil> has not been registered in stores

### DISCUSSION THREAD

**Discussion by golnar-m** [cosmos/community#49] [Chain Builders (Cosmos SDK)]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Hello,

I've created a custom module, when I want to use  functionalities of this module into another module, I'm facing this error:

**recovered: kv store with key <nil> has not been registered in stores**

I've already created kv store key but I got this bug.

Thank you

### Cosmos SDK Version

v0.45.5

**Comment by julienrbrt** [comment]
Hi, can you provide a snippet on how you are registering your module?
